src/components/uri/tests/Visualization.spec.tsx

```diff
@@ -713,6 +718,57 @@ describe("VisualizationWrapped", () => {
         });
     });
 
+    it("should add default sorting to the BarChart", async () => {
+        const props = {
+            sdk,
+            projectId,
+            uri: BARCHART_URI,
+            fetchVisObject,
+            fetchVisualizationClass,
+            uriResolver,
+            intl,
+            BaseChartComponent: BaseChart,
+        };
+
+        const wrapper = mount(<VisualizationWrapped {...props as any} />);
+        await testUtils.delay(FAST + 1);
+        wrapper.update();
+
+        const BaseChartElement = wrapper.find(BaseChart).get(0);
+
+        expect(BaseChartElement.props.resultSpec).toEqual({
+            dimensions: [
+                {
+                    itemIdentifiers: ["measureGroup"],
+                },
+                {
+                    itemIdentifiers: ["a1", "a2"],
+                },
+            ],
+            sorts: [
```

Comment:
why not to test only sorting? Who cares dimensions in this test case?

Reply:
removed